Lib/difflib.py

```diff
@@ -1111,6 +1112,9 @@ def unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
     For inputs that do not have trailing newlines, set the lineterm
     argument to "" so that the output will be uniformly newline free.
 
+    Set `color` to True to inject ANSI color codes and make the output look
```

Comment:
Arguments in italics, booleans in code formatting: https://devguide.python.org/documentation/markup/
```suggestion
    Set *color* to ``True`` to inject ANSI color codes and make the output look
```

Reply:
Done